src/io/flutter/devtools/DevToolsManager.java

```diff
@@ -224,16 +225,20 @@ public void processTerminated(@NotNull ProcessEvent event) {
   }
 
   public void openBrowserAndConnect(String serviceProtocolUri, String page) {
+    final String ideParam = "ide=" + (FlutterUtils.isAndroidStudio() ? "AndroidStudio&" : "IntelliJ&");
```

Comment:
why the trailing `&`?
I will instead joint the "ideParams" with '&'

Reply:
done.